docs/overview/welcome-to-the-thousand-brains-project-documentation.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Welcome to the Thousand Brains Project Documentation!
 ---
-
+  
```

Comment:
**issue**: Please don't add extra whitespace.

I found this VSCode extension to be helpful with trailing whitespace: https://marketplace.visualstudio.com/items?itemName=shardulm94.trailing-spaces